internal/services/contracts_events_consumer.go

```diff
@@ -145,3 +153,51 @@ func (c *ContractsEventsConsumer) setPrivilegeHandler(e *ContractEventData) erro
 
 	return nil
 }
+
+type MintedAftermarketDeviceArgs struct {
```

Comment:
It would feel good to grab this from the ABI but I don't think that'll play well with `mapstructure`.

Reply:
On that point, these field names don't look quite right.